website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -210,9 +284,9 @@ Here’s a table implementation that uses an array hash with localized strings f
 </Hds::Table>
 ```
 
-#### More Examples: replacing components as a pre-adoption step
+#### Replacing components as a pre-adoption step
 
-If you're not quite ready to replace your existing tables with this component, you can totally try out a pre-adoption spike with just the components themselves. It's a little more typing but it should give you an idea of what will work for you.
+If you’re not quite ready to replace your existing tables with this component, you can totally try out a pre-adoption spike with just the components themselves. It’s a little more typing but it should give you an idea of what will work for you.
```

Comment:
Two more considerations:
- the example below doesn't work (`@columns` is required, when there is a `@model`)
- I don't understand this example/paragraph. What makes this example a pre-adoption step? is using the `@model` (so is looping over records) and the yielded/contextual components.
Maybe you wanted to have an example where the table is generated using the non-yielded components? something like this?
```
<Hds::Table>
  <thead>
    <Hds::Table::Tr>
      <Hds::Table::Th>Artist</Hds::Table::Th>
      <Hds::Table::Th>Album</Hds::Table::Th>
      <Hds::Table::Th>Release Year</Hds::Table::Th>
    </Hds::Table::Tr>
  </thead>
  <tbody>
    {{#each this.myData as |record|}}
      <Hds::Table::Tr>
        <Hds::Table::Td>{{record.artist}}</Hds::Table::Td>
        <Hds::Table::Td>{{record.album}}</Hds::Table::Td>
        <Hds::Table::Td>{{record.year}}</Hds::Table::Td>
      </Hds::Table::Tr>
    {{/each}}
  </tbody>
</Hds::Table>
```

Reply:
Created a ticket for this: https://hashicorp.atlassian.net/browse/HDS-1585